src/Microsoft.DotNet.ProjectJsonMigration/ProjectDependencyFinder.cs

```diff
@@ -80,34 +80,23 @@ internal class ProjectDependencyFinder
                     projectContext.ProjectFile,
                     framework,
                     preResolvedProjects,
-                    solutionFile,
-                    HoistDependenciesThatAreNotDirectDependencies(projectToResolve, project)
+                    solutionFile
                 );
-                projects.AddRange(dependencies);
```

Comment:
Did you mean to remove this line? Seems like we still need to add dependencies right?